index.html

```diff
@@ -0,0 +1,231 @@
+        </p>
+  
+        <ul class="tag-ul">
+          <li> html </li>
+          <li> css</li>
+          <li> javascript</li>
+        </ul>
+  
+        <button type="button" class="btn-card">
+            See project 
+        </button>
+        </div>
+      
+    </div>
+
+   <!--Card three-->
+    <div class="card"> 
+      <div class='card-img'> 
+        <img src="./images/third_img.png" alt='project image'>
+      </div>
+      <div class="card-decri">
+        <h3>Multi-post <br /> Stories</h3>
+        <div class="tags-canopy">
+          <h5>CANOPY</h5>
+          <ul class="list-tags">
+            <li> Back End Dev </li>
+            <li>2015 </li>
+          </ul>
+        </div>
+        <p> A daily selection of privately
+            personalized reads; no account or sign-up required
+        </p>
+  
+        <ul class="tag-ul">
+          <li> html </li>
+          <li> css</li>
+          <li> javascript</li>
+        </ul>
+  
+        <button class="btn-card">
+            See project 
+        </button>
+        </div>
+      
+    </div>
+
+   <!--Card four-->
+    <div class="card"> 
+      <div class='card-img'> 
+        <img src="./images/forth_img.png" alt='project image'>
+      </div>
+      <div class="card-decri">
+        <h3>Multi-post <br /> Stories</h3>
+        <div class="tags-canopy">
+          <h5>CANOPY</h5>
+          <ul class="list-tags">
+            <li> Back End Dev </li>
+            <li>2015 </li>
+          </ul>
+        </div>
+        <p> A daily selection of privately
+            personalized reads; no account or sign-up required
+        </p>
+  
+        <ul class="tag-ul">
+          <li> html </li>
+          <li> css</li>
+          <li> javascript</li>
+        </ul>
+  
+        <button type="button" class="btn-card">
+            See project 
+        </button>
+        </div>
+      
+    </div>
+</section>
+  <!--End of my-section--> 
+        <!--ABOUT SECTION-->
+        <section class="about">
+          <div class="inner-about">
+            <h2 class="header">About <br> Myself</h2>
+            <p class="section">
+               I’m a software developer! I can help you build a product,feature or 
+               website Look through some of my work and experience! If you like what 
+               you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+           <p class="footer">LET CONNECT</p>
+             <ul class="imgs">
+              <li><a href="#"><img src="./images/twitter.svg" alt="twitter handle"></a></li>
+              <li><a href="#"><img src="./images/github.jpg" alt="github handle"></a></li>
+              <li><a href="#"><img src="./images/medium.jpg" alt="medium handle"></a></li>
+              <li><a href="#"><img src="./images/angellist.jpg" alt="angellisthandle"></a></li>
+             </ul>
+                <button class="btn-proj">Get my resume</button>
+          </div>
```

Comment:
I have noticed that there is a change of features in the  about section, the contents in the left block have been interchanged by those in the right block and this does not conform to how the design should look like. It would be nice if this is corrected 😄.

| Your design | Figma Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/145606795-a80242e2-1861-43e9-b3b3-675fcf1f937e.png) | ![image](https://user-images.githubusercontent.com/4247740/145606667-c7566506-0733-4ea2-b7fe-a305c8dc095e.png) |